EfEnumToLookupTests/Tests/TestStuff.cs

```diff
@@ -42,11 +48,41 @@ public void DoesStuff()
 				var actual = context.PeskyWabbits.First();
 				Assert.AreEqual("Roger", actual.Name);
 				Assert.AreEqual(Ears.Pointy, actual.TehEars);
+				Assert.AreEqual(Fur.Brown, actual.BodyFur);
 				Assert.AreEqual(1, context.PeskyWabbits.Count()); // spot unwanted re-use of db
 			}
 		}
 
 		[Test]
+		public void DoesStuffWithViews()
```

Comment:
thanks for expanding the tests to cover this, coverage is important to me on this library